ee/cis/macos-26/test/scripts/CIS_3.3_fail.sh

```
@@ -0,0 +1,22 @@
+      sub(/ttl=[0-9]+/, "ttl=30")
+    } else {
+      sub(/$/, " ttl=30")
+    }
+    if (!match($0, /all_max=/)) {
+      sub(/$/, " all_max=50M")
+    }
+    print
+    next
+  }
+  { print }
+' /etc/asl/com.apple.install > "$TMP"
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)